homeassistant/components/microbees/__init__.py

```diff
@@ -0,0 +1,51 @@
+""" Integration microBees """
+
+import logging
+from homeassistant.const import Platform, CONF_DOMAIN, CONF_PLATFORM
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from .servicesMicrobees import getBees
+
+_LOGGER = logging.getLogger(__name__)
+
+platforms = []
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up microBees from a config entry."""
+
+    token = dict(entry.data)["token"]
+
+    bees = await getBees(token)
+
+    entry.data = dict({"token": token, "bees": bees})
+
+    for bee in bees:
+        _LOGGER.info(bee)
+
+        for actuator in bee.get("actuators"):
+            match actuator.get("device_type"):
+                case 0:
+                    platforms.append(Platform.SWITCH)
+
+        match bee.get("productID"):
+            case 46:
+                platforms.append(Platform.SWITCH)
```

Comment:
Same thing here.

Reply:
we provide more platforms but for first pull request we removed the others platforms https://developers.home-assistant.io/docs/review-process/#home-assistant-core